packages/calcite-components/.stylelintrc.cjs

```diff
@@ -1,13 +1,7 @@
 // @ts-check
 
 // ⚠️ AUTO-GENERATED CODE - DO NOT EDIT
-const customFunctions = [
-  "get-trailing-text-input-padding",
-  "medium-modular-scale",
-  "modular-scale",
-  "scale-duration",
-  "small-modular-scale"
-];
+const customFunctions = [];
```

Comment:
Can we leave this file out?

Reply:
Whoops, yeah removed. This one keeps getting auto-generated at commit time.